.github/workflows/plugins-test.yaml

```diff
@@ -318,6 +318,8 @@ jobs:
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run redisson 3.x (37)
         run: bash test/plugin/run.sh redisson-scenario
+      - name: Run shardingsphere-3.x-scenario 3.0.0 (1)
```

Comment:
You miss the change of group name. `Redisson` -> `Redisson_ShardingSphere`

Reply:
Get it.